tests/test_imports.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+
+def test___all__(pytester: pytest.Pytester) -> None:
+    """See https://github.com/aio-libs/aiohttp/issues/6197"""
+    pytester.makepyfile(
+        test_a="""
+            from aiohttp import *
+        """
+    )
+    result = pytester.runpytest("-vv")
+    result.assert_outcomes(passed=0, errors=0)
+
+
+def test_web___all__(pytester: pytest.Pytester) -> None:
+    pytester.makepyfile(
+        test_b="""
+            from aiohttp.web import *
+        """
+    )
+    result = pytester.runpytest("-vv")
+    result.assert_outcomes(passed=0, errors=0)
+
+
+def test_import_time(pytester: pytest.Pytester) -> None:
+    """Check that importing aiohttp doesn't take too long.
+
+    Obviously, the time may vary on different machines and may need to be adjusted
+    from time to time, but this should provide an early warning if something is
+    added that significantly increases import time.
+    """
+    r = pytester.run("python3", "-c", "import aiohttp", timeout=0.3)
```

Comment:
Why not use a subprocess?
Also, instead of using some abstract-maybe-different interpreter, I'd rely on `sys.executable`.
Plus, it'd be nice to bake in `-Werror`.

Reply:
I was looking at the above tests as an example of how to run an external test, and then ended up looking at the other methods of `pytester`. But, I guess using `subprocess` would be just the same.

Reply:
Seems that the way `pytester` handles it is faster and more reliable (I think sending stderr/stdout to a temp file instead of PIPE may be one reason, although that doesn't make much sense to me). It seems safer to stick with this.